rln-js/index.html

```diff
@@ -258,23 +272,54 @@ <h4 class="mu1">Messages</h4>
 
       // Blockchain
 
-      generateCredsButton.onclick = () => {
-        membershipKey = rlnInstance.generateMembershipKey();
+      generateCredsButton.onclick = async () => {
+        // sign message with ethers
+        const signer = provider.getSigner();
+        const message =
+          "The signature of this message will be used to generate your RLN credentials. Ensure that this signature is only shared with the RLN dApp.";
```

Comment:
```suggestion
          "The signature of this message will be used to generate your RLN credentials. Anyone accessing it may send messages on your behalf, please only share with the RLN dApp";
```

Good to explain what may happen if you share.

Reply:
Fair enough; addressed